seed/building_sync/building_sync.py

```diff
@@ -365,15 +369,19 @@ def _parse_version(self):
         if self.element_tree is None:
             raise ParsingError('A file must first be imported with import method')
 
-        # first check if it's a file form Audit Template Tool and infer the version
-        # Currently ATT doesn't include a schemaLocation so this is necessary
-        if self._is_from_audit_template_tool():
-            return self.BUILDINGSYNC_V2_0
-
         bsync_element = self.element_tree.getroot()
         if not bsync_element.tag.endswith('BuildingSync'):
             raise ParsingError('Expected BuildingSync element as root element in xml')
 
+        # first check for a version attribute in the buldingsync tag
+        if "version" in bsync_element.attrib:
+            return bsync_element.attrib["version"]
+
+        # second check if it's a file form Audit Template Tool and infer the version
+        # Currently ATT doesn't include a schemaLocation so this is necessary
+        if self._is_from_audit_template_tool():
+            return self.BUILDINGSYNC_V_LATEST
```

Comment:
Ah I didn't think about this, but if a file is missing the version attribute and it appears to be from ATT, then we should assume the bsync version is `2.0`, not the latest (v2.0 was the last version which didn't require a version attribute). Could you update the comment to say something like
```
# second check if it's a file form Audit Template Tool
# if so, then it must be a 2.0 file b/c that was the last version which didn't require @version
```

Reply:
gotcha!